src/lib.rs

```diff
@@ -42,7 +34,7 @@ pub use graph::{Module, parse as graph_parse, generate as graph_generate};
 pub use ref_list::{RefList, Entry, EntryRef, DeleteTransaction};
 #[cfg(feature = "std")]
 pub use export_globals::export_mutable_globals;
-pub use parity_wasm::elements::Instruction;
+pub use parity_wasm;
```

Comment:
maybe just `pub use parity_wasm::elements` ?

Reply:
I think we want the complete thing. My plan is to never have a crate to directly depend on `parity-wasm` when it also uses `pwasm-utils`. Imagine some crate which uses `parity_wasm::builder` to crate some module and then want to pass it into `pwasm-utils`. This would still need to depend on both and keep them in sync. I want those crates to just have to depend on `pwasm-utils`.